tests/tests.sh

```diff
@@ -158,4 +144,5 @@ function init_tests {
     if [[ $FAILURE = 1 ]]; then
         exit $FAILURE
     fi
+    test_websocket
```

Comment:
You moved it after script exits? It'll never run. Well that's certainly not what I meant when I said make it the last test! :joy: